src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx

```diff
@@ -79,11 +82,38 @@ tuner.search(
     {
       title: 'Integrate XGBoost',
       docsLink: DOCUMENTATIONS.INTEGRATIONS.XGBOOST,
-      code: `from aim.xgboost import AimCallback
+      code: `from sklearn import datasets
+from sklearn.model_selection import train_test_split
+import xgboost as xgb
+
+
+from mlflow import set_tracking_uri, start_run
+import mlflow.xgboost
+
+# Set FastTrackML tracking server
+set_tracking_uri("${fasttrack_server}")
```

Comment:
i think this one should use `mlflow` module prefix so it's clearer where it comes from -- maybe just import all of mlflow for that?